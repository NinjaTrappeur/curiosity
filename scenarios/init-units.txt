unit create alpha Alpha
unit update alpha "The first business unit of the prototype, Alpha is run by @mila."
