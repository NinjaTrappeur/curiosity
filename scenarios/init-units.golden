1: business create alpha Alpha
Business entity created: BENT-1
